source/partials/_progress-bars.html.haml

```diff
@@ -0,0 +1,5 @@
+%h5.progress-header= language.name
+.progress
+  .progress-wrap{ style: "width: #{progress_bar language};" }
```

Comment:
Here again: name is confusing. My suggestion: `percentage_of language.lessons.complete`